sdk/ml/azure-ai-ml/azure/ai/ml/entities/_inputs_outputs/external_data.py

```diff
@@ -37,27 +35,32 @@ def __init__(
 class Database(DictMixin, RestTranslatableMixin):  # pylint: disable=too-many-instance-attributes
     """Define a database class for a DataTransfer Component or Job.
 
-    e.g.
-    source_snowflake = Database(query='SELECT * FROM my_table', connection='azureml:my_azuresql_connection')
-    or
-    stored_procedure_params = [{'name': 'job', 'value': 'Engineer', 'type': 'String'},
-    {'name': 'department', 'value': 'Engineering', 'type': 'String'}]
-    source_snowflake = Database(stored_procedure='SelectEmployeeByJobAndDepartment',
-                                stored_procedure_params=stored_procedure_params,
-                                connection='azureml:my_azuresql_connection')
-
-    :param query: The sql query to get data from database
-    :type query: str
-    :param table_name: The database table name
-    :type table_name: str
-    :param stored_procedure: stored_procedure
-    :type stored_procedure: str
-    :param stored_procedure_params: stored_procedure_params
-    :type stored_procedure_params: List[dict, StoredProcedureParameter]
-    :param connection: Connection is workspace, we didn't support storage connection here, need leverage workspace
-    connection to store these credential info.
-    :type connection: str
-    :raises ~azure.ai.ml.exceptions.ValidationException: Raised if Source cannot be successfully validated.
+    Example usage:
+    - For querying a database table:
```

Comment:
Could you move this into samples and reference it like the inline examples? If it's in the docstring in plaintext like this, it won't be tested.

Reply:
done. 
But this class is actually private, isn't it good for private classes to write samples and publish them to users?